ffi/include/tvm/ffi/container/ndarray.h

```diff
@@ -202,10 +219,6 @@ class NDArrayObjFromDLPack : public NDArrayObj {
  public:
   explicit NDArrayObjFromDLPack(TDLPackManagedTensor* tensor) : tensor_(tensor) {
     *static_cast<DLTensor*>(this) = tensor_->dl_tensor;
-    // set strides to nullptr if the tensor is contiguous.
-    if (IsContiguous(tensor->dl_tensor)) {
```

Comment:
also need to check `tensor->dl_tensor->strides == nullptr` and act accordingly if needed